src/components/LearningCard.js

```diff
@@ -0,0 +1,88 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+import ButtonExpansion from './ButtonExpansion'
+import { useState } from 'react'
+import GlobalStyle from '../GlobalStyle'
+
+Card.propTypes = {
+  onClick: PropTypes.func,
+  isExpanded: PropTypes.bool,
+  name: PropTypes.string,
+  description: PropTypes.string,
+  tags: PropTypes.array,
+  color: PropTypes.string,
```

Comment:
The props for your card do not include `onClick` and `isExpanded`.
```suggestion
  name: PropTypes.string,
  description: PropTypes.string,
  tags: PropTypes.array,
  color: PropTypes.string,
```